### DISCUSSION TITLE
Vercel deployment with database fails with SIGKILL error

### DISCUSSION THREAD

**Discussion by ronald-willems** [lobehub/lobehub#4475] [Self Hosting] [answered]
I'm trying to set up self hosting with database server, following this guide: (https://lobehub.com/docs/self-hosting/server-database/vercel).

The deployment now fails with SIGKILL error which seems to be a memory constraint problem. Has anyone overcome this issue?

Part of the log below

Creating an optimized production build ...
 ○ (serwist) The service worker will not be automatically registered, please call 'window.serwist.register()' in 'componentDidMount' or 'useEffect'.
 ✓ (serwist) Bundling the service worker script with the URL '/sw.js' and the scope '/'...
 ✓ Compiled successfully
   Linting and checking validity of types ...
error: script "build" was terminated by signal SIGKILL (Forced quit)
Error: Command "bun run build" exited with SIGKILL
▲ Build system report
▲ To always completely log this report, add VERCEL_BUILD_SYSTEM_REPORT=1 as an Environment Variable to your project.
• At least one "Out of Memory" ("OOM") event was detected during the build.
  • This occurs when processes or applications running during the build completely fill up the available memory (RAM) in the build container. When this happens, the build container terminates one of the processes during the build with a SIGKILL signal.
  • Read this troubleshooting guide for more information: https://vercel.link/troubleshoot-build-errors

**Comment by arvinxx** [comment, marked as answer]
https://github.com/lobehub/lobe-chat/issues/4471
